controller/src/bin/xcvradm.rs

```diff
@@ -13,29 +20,145 @@ async fn request_handler(_: SpRpcRequest) -> Result<HostRpcResponse, Error> {
     Err(Error::Protocol(MessageError::ProtocolError))
 }
 
+fn parse_xvcr_list(s: &str) -> Result<Vec<u8>, String> {
+    s.split(",")
+        .map(|x| {
+            x.parse()
+                .map_err(|_| String::from("invalid transceiver index"))
+        })
+        .collect()
+}
+
+fn parse_log_level(s: &str) -> Result<Level, String> {
+    s.parse().map_err(|_| String::from("invalid log level"))
+}
+
+#[derive(Parser)]
+struct Args {
+    #[command(subcommand)]
+    cmd: Cmd,
+
+    /// The FPGA whose transceivers to address.
+    #[arg(short, long, default_value_t = 0)]
+    fpga_id: u8,
+
+    /// The comma-separated list of transcievers on the FPGA to address.
+    ///
+    /// The default is all transceivers on an FPGA.
+    #[arg(short, long)]
+    transceivers: Option<String>,
```

Comment:
On second thought, I think we can do the parsing here (instead of in `parse_xvcr_list`):
```suggestion
    #[clap(long, short, use_value_delimiter = true)]
    transceivers: Option<Vec<u8>>,
```

Reply:
TIL about `use_value_delimiter`! 😆